views/layout.hbs

```diff
@@ -1,19 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>{{title}}</title>
-    <link rel="stylesheet" href="/stylesheets/style.css" />
-  </head>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>{{title}}</title>
+  <link rel="stylesheet" href="/stylesheets/style.css" />
```

Comment:
I just want to mention that it's usually better practice to put your custom stylesheet after the bootstrap stylesheet to avoid your styles getting overwritten. 

You can check [this link](https://stackoverflow.com/questions/33202459/bootstrap-and-custom-css-loading-order-misunderstandings) for a more detailed explanation

Reply:
Thanks for the tip Lloyd :)